spec/models/item_spec.rb

```diff
@@ -2,12 +2,16 @@
 
 RSpec.describe Item, type: :model do
 
+  let!(:item) {FactoryGirl.create(:item)}
+  let(:item2) {FactoryGirl.create(:item, price: 3,
+              created_at: Date.today, shelf_life_days: -1)}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 7, 59 should be on its own line.